packages/app/.prettierrc

```diff
@@ -6,5 +6,6 @@
   "singleQuote": true,
   "trailingComma": "es5",
   "bracketSpacing": true,
-  "jsxBracketSameLine": false
+  "jsxBracketSameLine": false,
+  "arrowParens": "avoid"
```

Comment:
Prettier 2.0 [sets the default](https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-arrowparens-to-always-7430httpsgithubcomprettierprettierpull7430-by-kachkaevhttpsgithubcomkachkaev) to `"always"`. I think we can continue using `"always"` since it makes it easier to add types in TypeScript (when we need to).

You can then use `npx prettier -w components/**/*.js *.js` to update all files with the new setting

Reply:
ESLint uses the Prettier configuration that's why the linting is failing